### ISSUE TITLE
Friends List refusing to load with error 101-0112

### ISSUE THREAD

**Issue by ResourceName** [PretendoNetwork/friends#12]
When attempting to load the friends list, it takes a long time then spews out error 101-0112.

Tested Nintendo's Friend List and it works fine.

**Comment by Breezygames** [comment]
I get this issue too on 3DS and then #13 

Edit: The issue seems to be fixed. (at least on 3ds)

**Comment by ted-mundy** [comment]
should this issue be closed?

**Comment by jonbarrow** [comment]
> should this issue be closed?

No, this is likely related to https://github.com/PretendoNetwork/nex-go/issues/53

**Comment by JPeisach** [comment]
There certainly appears to be less cases of this. In the support thread on discord, the most recent report of this was 5 days ago, the 2nd recent 24 days ago.
